app/models/concerns/audit_associations.rb

```diff
@@ -0,0 +1,42 @@
+module AuditAssociations
+  def changes_to_save
+    unless audited_options[:associations].blank?
```

Comment:
it would be better to write `if audited_options[:associations].present` since there's else branch